libshaderc_util/include/libshaderc_util/compiler.h

```diff
@@ -111,7 +121,7 @@ class Compiler {
         suppress_warnings_(false),
```

Comment:
Alternately, there could be a Default = Vulkan value  enum class TargetEnv.  Then it would be more clear.
But I like forcing the callers to think about what they mean.
So I think I want to avoid a Default value.